doctoshotgun.py

```diff
@@ -455,25 +460,50 @@ def try_to_book_place(self, profile_id, motive_id, practice_id, agenda_ids, vac_
 class DoctolibDE(Doctolib):
     BASEURL = 'https://www.doctolib.de'
     KEY_PFIZER = '6768'
+    KEY_PFIZER_SECOND = '6769'
+    KEY_PFIZER_THIRD = None
     KEY_MODERNA = '6936'
+    KEY_MODERNA_SECOND = '6937'
+    KEY_MODERNA_THIRD = None
     KEY_JANSSEN = '7978'
+    KEY_ASTRAZENECA = '7109'
+    KEY_ASTRAZENECA_SECOND = '7110'
+
     vaccine_motives = {
         KEY_PFIZER: 'Pfizer',
+        KEY_PFIZER_SECOND: 'Zweit.*Pfizer|Pfizer.*Zweit',
+        KEY_PFIZER_THIRD: 'Dritt.*Pfizer|Pfizer.*Dritt',
         KEY_MODERNA: 'Moderna',
+        KEY_MODERNA_SECOND: 'Zweit.*Moderna|Moderna.*Zweit',
+        KEY_MODERNA_THIRD: 'Dritt.*Moderna|Moderna.*Dritt',
         KEY_JANSSEN: 'Janssen',
+        KEY_ASTRAZENECA: 'AstraZeneca',
+        KEY_ASTRAZENECA_SECOND: 'Zweit.*AstraZeneca|AstraZeneca.*Zweit'
     }
     centers = URL(r'/impfung-covid-19-corona/(?P<where>\w+)', CentersPage)
     center = URL(r'/praxis/.*', CenterPage)
 
 class DoctolibFR(Doctolib):
     BASEURL = 'https://www.doctolib.fr'
     KEY_PFIZER = '6970'
+    KEY_PFIZER_SECOND = '6971'
+    KEY_PFIZER_THIRD = '8192'
     KEY_MODERNA = '7005'
+    KEY_MODERNA_SECOND = '7004'
+    KEY_MODERNA_THIRD = '8193'
     KEY_JANSSEN = '7945'
+    KEY_ASTRAZENECA = '7107'
+    KEY_ASTRAZENECA_SECOND = '7108'
     vaccine_motives = {
-        KEY_PFIZER: 'Pfizer',
-        KEY_MODERNA: 'Moderna',
+        KEY_PFIZER: '1re.*Pfizer',
+        KEY_PFIZER_SECOND: '2re.*Pfizer',
+        KEY_PFIZER_THIRD: '3de.*Pfizer',
+        KEY_MODERNA: '1re.*Moderna',
+        KEY_MODERNA_SECOND: '2re.*Moderna',
+        KEY_MODERNA_THIRD: '3de.*Moderna',
         KEY_JANSSEN: 'Janssen',
+        KEY_ASTRAZENECA: '1re.*AstraZeneca',
+        KEY_ASTRAZENECA_SECOND: '2de.*AstraZeneca'
```

Comment:
Will work only in France I guess?

Reply:
Yes, i guess it should be changed to "AstraZeneca". Otherwise the vaccine would not be found when there is no additional e. g. second and/or third shot.